Indicators/KaufmanAdaptiveMovingAverage.cs

```diff
@@ -116,10 +119,11 @@ protected override decimal ComputeNextValue(IReadOnlyWindow<IndicatorDataPoint>
             _trailingValue = newTrailingValue.Value;
 
             // Calculate the efficiency ratio
-            var efficiencyRatio = (_sumRoc1 <= _periodRoc) || _sumRoc1 == 0 ? 1m : Math.Abs(_periodRoc / _sumRoc1);
+            var kefValue = (_sumRoc1 <= _periodRoc) || _sumRoc1 == 0 ? 1m : Math.Abs(_periodRoc / _sumRoc1);
```

Comment:
We don't need to change the variable name here since `efficiencyRatio` describes well what we are calculating.